src/pages/why.tsx

```diff
@@ -93,13 +93,13 @@ export default function Home() {
                         Bible in a deeper way.
                     </p>
                     <hr className="mx-0 w-full border-t-2  border-black" />
-                    <h2>Why meditate on God&apos;s word?</h2>
+                    <h2 className="dark:text-white">Why meditate on God&apos;s word?</h2>
                     <p>
                         In the process of making typetheword.site, these
                         scriptures were great reminders of why we should
                         meditate on God&apos;s word. If you have others, feel
                         free to{' '}
-                        <EmailLink className="underline">email me</EmailLink>. I
+                        <EmailLink className="underline dark:text-white">email me</EmailLink>. I
```

Comment:
I think this change will be used in all usages of `EmailLink` so I would prefer to have this within the component text itself.